src/App.tsx

```diff
@@ -1,12 +1,21 @@
-import React from 'react';
-import './App.css';
+import React, { Fragment, lazy, Suspense } from 'react';
+import { Switch, Route } from 'react-router-dom';
+import './assets/app.scss';
+import ErrorBoundary from './components/errors/errorBoundary/errorBoundary';
```

Comment:
I am assuming importing our components will be pretty long. We can use a path resolution system to help keep it clean and consistent.

Reply:
Its just for information sake